sync/src/main/java/tech/pegasys/artemis/sync/PendingPool.java

```diff
@@ -116,16 +123,23 @@ public void add(T item) {
     }
 
     final Bytes32 itemRoot = hashTreeRootFunction.apply(item);
-    final Collection<Bytes32> dependentBlockRoots = dependentBlockHashFunction.apply(item);
+    final Collection<Bytes32> requiredRoots = requiredBlockRootsFunction.apply(item);
 
-    dependentBlockRoots.forEach(
-        dependentBlockRoot ->
-            // Index block by parent
-            pendingItemsByDependentBlockRoot
+    requiredRoots.forEach(
+        requiredRoot ->
+            // Index item by required roots
+            pendingItemsByRequiredBlockRoot
```

Comment:
(nit) I wound consider renaming `pendingItemsByRequiredBlockRoot` to `requiredBlockRootToPendingItems`. It took me a loooong while to figure out